template/default.nix

```diff
@@ -29,6 +29,7 @@ rec {
     tag = dockerTag;
     contents = [ pkgs.bashInteractive pkgs.coreutils pkgs.util-linuxMinimal ];
     config = {
+      Env = ["PRODUCT_CONFIG=${deploy/config-spec/properties.yaml}"];
```

Comment:
Hm, we might need to think a bit deeper about this before adding it, not all of our operators have product-configs at all.

Reply:
One way would be to make it conditional on properties.yaml existing:

```nix
Env = lib.optional (builtins.pathExists deploy/config-spec/properties.yaml) "PRODUCT_CONFIG=${deploy/config-spec/properties.yaml}";
```

Reply:
done